x/oracle/internal/keeper/task.go

```diff
@@ -151,7 +153,8 @@ func (k Keeper) GetAllTasks(ctx sdk.Context) (tasks []types.Task) {
 // UpdateAndGetAllTasks updates all tasks and returns them.
 func (k Keeper) UpdateAndGetAllTasks(ctx sdk.Context) (tasks []types.Task) {
 	k.IteratorAllTasks(ctx, func(task types.Task) bool {
-		_ = k.UpdateWaitingBlocks(ctx, task)
+		// _ = k.UpdateWaitingBlocks(ctx, task)
```

Comment:
I guess this function can be removed?

Reply:
I think so. It seems this function is not used elsewhere.